### ISSUE TITLE
Unexpected errors with Ansible Lint in pycharm and WSL

### ISSUE THREAD

**Issue by teodoryantcheff** [4ch1m/intellij-ansible-lint#26]
I have never managed to get it to actually "lint". Here are all the tech details that I thought maybe pertinent : 

Ansible (in the WSL2 VM):
```shell
teo@jailbreaker-pc:/mnt/e/src/ansible$ ansible --version
ansible [core 2.18.2]
  config file = /mnt/e/src/ansible/ansible.cfg
  configured module search path = ['/home/teo/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/teo/.local/pipx/venvs/ansible/lib/python3.12/site-packages/ansible
  ansible collection location = /home/teo/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/teo/.local/bin/ansible
  python version = 3.12.3 (main, Feb  4 2025, 14:48:35) [GCC 13.3.0] (/home/teo/.local/pipx/venvs/ansible/bin/python)
  jinja version = 3.1.5
  libyaml = True
teo@jailbreaker-pc:/mnt/e/src/ansible$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 24.04.2 LTS
Release:        24.04
Codename:       noble
teo@jailbreaker-pc:/mnt/e/src/ansible$ uname -a
Linux jailbreaker-pc 5.15.167.4-microsoft-standard-WSL2 #1 SMP Tue Nov 5 00:21:55 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
teo@jailbreaker-pc:/mnt/e/src/ansible$ ansible-lint --version
ansible-lint 25.1.3 using ansible-core:2.18.2 ansible-compat:25.1.4 ruamel-yaml:0.18.10 ruamel-yaml-clib:0.2.12
teo@jailbreaker-pc:/mnt/e/src/ansible$
```

WSL:
```powershell
C:\Users\Teo>wsl --version
WSL version: 2.4.11.0
Kernel version: 5.15.167.4-1
WSLg version: 1.0.65
MSRDC version: 1.2.5716
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.26100.1-240331-1435.ge-release
Windows version: 10.0.19045.5487

C:\Users\Teo>wsl -l
Windows Subsystem for Linux Distributions:
Ubuntu-20.04 (Default)
docker-desktop

C:\Users\Teo>
```
Running under Windows 10 pro.

PyCharm:
```
PyCharm 2024.3.3
Build #PY-243.24978.54, built on February 12, 2025
Runtime version: 21.0.6+8-b631.39 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
```

Plugin configuration:
![Image](https://github.com/user-attachments/assets/a2c5c4c8-161e-42c8-9e96-3ec1d6599179)

The error I get when TEST is clicked:
![Image](https://github.com/user-attachments/assets/e2f00187-7f5f-45de-ad6c-3fe191d6c3cc)

Whenever a `.yml` file is being edited, these errors pop up in the lower right corner of pycharm:
![Image](https://github.com/user-attachments/assets/950a5b5d-4494-44bf-adda-f01fd30544e0)

Plugin version 1.10.1.

What am I doing wrong?

**Comment by 4ch1m** [comment]
The plugin sends detailed (exception-)information to the [IDE-log](https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files).

Do you find anything related there?

**Comment by teodoryantcheff** [comment]
🤦 

```
java.lang.Throwable: Unexpected exit code from 'ansible-lint': 127; |/bin/bash: line 1: ansible-lint: command not found
```

I have `ansible-lint` installed in the container with `pipx`, via:

```shell
pipx install --include-deps ansible
``` 

What do I need to do to make it invokable from Ansilble Lint ? 

And, of course, thank you for taking the time to look into this!

**Comment by 4ch1m** [comment]
I'm sorry... I can't give support for your (WSL-)environment.

Please check the [README](https://github.com/4ch1m/intellij-ansible-lint?tab=readme-ov-file#requirements):

```
ansible-lint needs to be runnable directly from your computer's command line.
```

Just make sure that `ansible-lint` is generally callable within your environment (-> `$PATH`).
You could also try to set the **full/absolute** path to `ansible-lint` in the plugin settings.

**Comment by teodoryantcheff** [comment]
![Image](https://github.com/user-attachments/assets/fa0e7fdc-99e7-4802-97d4-1e9e9886ea4f)

Thank you !
